README.md

```diff
@@ -1,28 +1,195 @@
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+See the support scope for [Red Hat Technology Preview](https://access.redhat.com/support/offerings/techpreview/) features for more details.
 
-## Overview
 
-The automation within this repository establishes the components of Trustification, the downstream redistribution of [Trustification project](https://github.com/trustification/trustification) within a single Red Hat Enterprise Linux (RHEL) or Fedora machine using a standalone containerized deployment. Containers are spawned using Kubernetes based manifests using
-[podman kube play](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
+## Description
+
+The RHTPA service is the downstream redistribution of the [Trustification](https://github.com/trustification/trustification) project.
+The automation contained within this Git repository installs and configures the components of RHTPA to run on a single RHEL server, which uses a standalone containerized deployment.
+A Kubernetes-based manifest creates containers that uses [`podman kube play`](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
 
-The following Trustification components are deployed as part of this architecture:
+The RHTPA Ansible collection deploys the following RHTAS components:
 
 - [Trustification](https://github.com/trustification/trustification)
 - [Guac](https://github.com/trustification/guac)
 
-The following components are used if provided by the customers:
+An [NGINX](https://www.nginx.com) front end places an entrypoint to the RHTPA UI.
+A set of self-signed certificates get generated at runtime to establishing secure communications.
+
+The ingress host name is follow, where `<base_hostname>` is your deployment's base hostname:
+* https://`<base_hostname>`
+
+## Requirements
+
+* Ansible 2.16.0 or greater
+* Python 3.9.0 or greater
+* RHEL x86\_64 9.2 or greater.
+* Installation and configuration of Ansible on a control node to perform the automation.
+* Installation of the Ansible collections on the control node.
+  * If installing from the Ansible Automation Hub, then run `ansible-galaxy install redhat.trusted_profile_analyzer`.
+  * If installing from this Git repository, then clone it locally, and run `ansible-galaxy collection install -r requirements.yml`.
+* An OpenID Connect (OIDC) provider, such as [Keycloak](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/).
+* A PostgreSQL instance
+* SQS like [Kafka](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_streams/)
+* S3 service or S3 compatible service
+* Optional:
+  Installation of the `podman` binaries to verify that the RHTPA service is working as expected.
+
+## Overview
+The following components are provided by the customers:
+
+### RH Single Sign On
```

Comment:
We should not use `RH` to abbreviate Red Hat, please change.

`RH` --> `Red Hat`